stubs/pyxdg/xdg/BaseDirectory.pyi

```diff
@@ -8,11 +8,11 @@ xdg_config_dirs: list[str]
 xdg_cache_home: str
 xdg_state_home: str
 
-def save_config_path(*resource: PathLike[str]) -> str: ...
-def save_data_path(*resource: PathLike[str]) -> str: ...
-def save_cache_path(*resource: PathLike[str]) -> str: ...
-def save_state_path(*resource: PathLike[str]) -> str: ...
-def load_config_paths(*resource: PathLike[str]) -> Iterator[str]: ...
-def load_first_config(*resource: PathLike[str]) -> str: ...
-def load_data_paths(*resource: PathLike[str]) -> Iterator[str]: ...
+def save_config_path(*resource: PathLike[str] | str) -> str: ...
+def save_data_path(*resource: PathLike[str] | str) -> str: ...
+def save_cache_path(*resource: PathLike[str] | str) -> str: ...
+def save_state_path(*resource: PathLike[str] | str) -> str: ...
+def load_config_paths(*resource: PathLike[str] | str) -> Iterator[str]: ...
+def load_first_config(*resource: PathLike[str] | str) -> str: ...
+def load_data_paths(*resource: PathLike[str] | str) -> Iterator[str]: ...
```

Comment:
Use our `_typeshed.StrPath` alias for cases like these:

https://github.com/python/typeshed/blob/ecf4e708bff2f99d3297f801e52f0ba1d7ca0436/stdlib/_typeshed/__init__.pyi#L134